common/djangoapps/entitlements/tests/factories.py

```diff
@@ -15,6 +15,9 @@ class Meta(object):
 
     uuid = factory.LazyFunction(uuid4)
     course_uuid = factory.LazyFunction(uuid4)
-    mode = FuzzyChoice([CourseMode.VERIFIED, CourseMode.PROFESSIONAL])
+    mode = FuzzyChoice(
+        [CourseMode.VERIFIED, CourseMode.PROFESSIONAL, CourseMode.NO_ID_PROFESSIONAL_MODE, CourseMode.CREDIT_MODE]
+    )
+    mode = FuzzyChoice(['professional', 'verified', 'no-id-professional', 'credit'])
```

Comment:
It looks like one of these definitions for `mode` could be removed.

Also, are credit entitlements valid?